core/src/main/java/discord4j/core/object/entity/channel/ForumChannel.java

```diff
@@ -0,0 +1,247 @@
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Gets the default emoji to add in a reaction button on a thread for this forum channel
+     *
+     * @return An {@link Optional} that may contain the {@link DefaultReaction} emoji
+     */
+    public Optional<DefaultReaction> getDefaultReaction() {
+        return getData().defaultReactionEmoji()
+            .toOptional().orElseThrow(IllegalStateException::new) // Mandatory for Forum channels
+            .map(defaultReactionData -> new DefaultReaction(getClient(), defaultReactionData));
+    }
+
+    /**
+     * Gets the default thread rate limit duration per user.
+     * This field is copied into a created thread and is not updated on the fly.
+     *
+     * @return The initial rate limit per user for newly created threads
+     */
+    public int getDefaultThreadRateLimitPerUser() {
+        return getData().defaultThreadRateLimitPerUser().toOptional()
+            .orElseThrow(IllegalStateException::new); // Mandatory for Forum channels
+    }
+
+    @Experimental
+    public enum SortOrder {
+
+        /**
+         * Internal value only, for unknown values
+         */
+        UNKNOWN(-1),
+        LATEST_ACTIVITY(0),
+        CREATION_DATE(1);
+
+        /**
+         * Discord represented integer value
+         */
+        private final int value;
+
+        SortOrder(int value) {
+            this.value = value;
+        }
+
+        /**
+         * Gets Discord represented value
+         *
+         * @return Discord integer represented value
+         */
+        public int getValue() {
+            return value;
+        }
+
+        /**
+         * Translates an integer value to a {@link SortOrder}
+         *
+         * @param value The integer value
+         * @return The SortOrder if known, or the fallback UNKNOWN value
+         */
+        public static SortOrder valueOf(final int value) {
+            /* We could exclude UNKNOWN from the lookup array using
+            Arrays.copyOfRange(SortOrder.values(), 1, SortOrder.values().length) but this would just waste CPU time
+            because if -1 is one day used by Discord we would need anyway to update the code here */
+            for (SortOrder sortOrder : SortOrder.values()) {
+                if (sortOrder.getValue() == value) {
+                    return sortOrder;
+                }
+            }
+            // Fallback to UNKNOWN as default value
+            return SortOrder.UNKNOWN;
+        }
+    }
+
+    /**
+     * Gets the default associated {@link SortOrder} for this forum channel
+     *
+     * @return The {@link SortOrder}, wrapped in an {@link Optional}
+     */
+    public Optional<SortOrder> getSortOrder() {
+        return getData().defaultSortOrder().toOptional()
+            .orElseThrow(IllegalStateException::new) // Mandatory for Forum channels
+            .map(SortOrder::valueOf);
+    }
+
+    @Experimental
+    public enum LayoutType {
```

Comment:
I think is better move this enum to the final for keep methods upper.

Reply:
Done in next commit